src/test/groovy/com/onesignal/androidsdk/MainTest.groovy

```diff
@@ -318,13 +318,13 @@ class MainTest extends Specification {
     def "Upgrade to compatible OneSignal SDK when firebase-iid:20.1.6 is used"() {
         when:
         def results = runGradleProject([
-                'android.useAndroidX': true,
-                'android.enableJetifier': true,
-                compileLines : """\
+            'android.useAndroidX': true,
+            'android.enableJetifier': true,
```

Comment:
Do we need jetifier enabled if AndroidX is being used?

Reply:
For a normal full build yes, we need to Jetifier for our SDK to work at runtime.
However since the dependencies are only being checked we don't need `android.enableJetifier` here so removing to make this test simpler.

Reply:
Removed